ex5_ch2.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jun 13 10:08:53 2018
+
+@author: amyhaddad
+"""
+
+import math 
+
+#step1: write input statement asking for one number
+first_number = int(input("This is your first number: "))
+
```

Comment:
Question: Based on my question, I'm assuming the user inputs a number. However, what if they write out a number, like "seven." How do I convert the string to an integer if the answer will always be different each time?

Reply:
You have to make the assumption that they will type it in correctly. Down the road, you'll learn ways to catch bad input and tell the user to enter it in differently. But for now, I'd just focus on getting the program to work properly with good input. 

So this line is exactly how I'd do it!